ui-v2/src/components/schemas/readme.md

```diff
@@ -45,26 +45,33 @@ The schema form uses the `PrefectSchemaObject` type to define the schema.
 ## Prefect Kind Values
 A value can dictate the type of input that is rendered by using the `__prefect_kind` property. Using this property, the form will know how to render the value. For this example a json input will be rendered rather than the default object input.
 
-```ts
-const schema = {
-  type: "object",
-  properties: {
-    user: {
-      type: "object",
-      properties: {
-        name: { type: "string", title: "Name" },
-      },
-    },
-  },
-};
+```tsx
+import { SchemaForm, useSchemaForm, type PrefectSchemaObject } from "@/components/schemas";
 
-const values = {
-  user: {
-    name: {
-      __prefect_kind: "json",
-      value: "John Doe",
-    },
-  },
+const schema = {
+	type: "object",
+	properties: {
+		name: { type: "string", title: "Name" },
+	},
+} satisfies PrefectSchemaObject;
```

Comment:
(Preference)

There's almost never a reason to use `satisfies` unless we're trying to preserve the inferred type. For things like this we should just use the type as is which I think is easier to grok personally. 

```suggestion
const schema: PrefectSchemaObject = {
	type: "object",
	properties: {
		name: { type: "string", title: "Name" },
	},
};
```